app/models/miq_schedule_worker/runner.rb

```diff
@@ -329,20 +324,19 @@ def schedules_for_ems_metrics_coordinator_role
 
   def schedules_for_event_role
     # These schedules need to run by the servers with the event role
-    return unless @active_roles.include?("event")
-    @schedules[:event] ||= []
-
+    return unless schedule_enabled?(:event)
+    s = scheduler_for(:event)
```

Comment:
Again, I"m not a fan of `s` here, it's short but we lose the context of what role we're on here as you page down in each of these methods.